### ISSUE TITLE
[Feature] Prefixes 2/n: Ignored glob prefixes

### ISSUE THREAD

**Issue by olets** [olets/zsh-abbr#135]
Issue for documentation/findability.

- For discussion, see #114

See also

- #134
- #136 

This feature is current a candidate for v6.

- There is an array of regular abbreviation "prefixes" which are globs. These are ignored when checking for regular abbreviations
- Default is an empty array
- User can customize the array

After:

```
# .zshrc
the_glob_prefixes_array_name_tbd=( '?.' )
```

```
% abbr b=cde
% a.b[SPACE] # expands to a.cde
```

**Comment by olets** [comment]
This is in v6.0.0-beta.1

**Comment by olets** [comment]
Released in v6.0.0
